ranger/ext/img_display.py

```diff
@@ -898,6 +934,57 @@ def _format_cmd_str(self, cmd, payload=None, max_slice_len=2048):
         else:
             yield self.protocol_start + central_blk + b';' + self.protocol_end
 
+    # Lazily load and parse diacritics use to index row and column from
+    # rowcolumn-diacritics.txt
+    def _get_diacritics(self):
+        if (self.diacritics is not None):
+            return self.diacritics
+
+        try:
+            with open(
+                self.fm.relpath("rowcolumn-diacritics.txt"),
```

Comment:
```suggestion
                self.fm.relpath(“data”, “rowcolumn-diacritics.txt"),
```
@Ethsan you placed file in data folder, so the path is incorrect and throws error.
also why file named txt while it’s treated like csv? 🤔

Reply:
Hi,
> @Ethsan you placed file in data folder, so the path is incorrect and throws error.

Yeah sorry about that, I move it will trying ot fix linting issues and forgot to update path.

> also why file named txt while it’s treated like csv? 🤔

I have simply reused the name given upstream [rowcolumn-diacritics.txt](https://github.com/kovidgoyal/kitty/blob/master/gen/rowcolumn-diacritics.txt)